src/OrderImport/Rules/ManomanoMissingCarrier.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ *  Copyright since 2019 Shopping Feed
+ *
+ *  NOTICE OF LICENSE
+ *
+ *  This source file is subject to the Academic Free License (AFL 3.0)
+ *  that is bundled with this package in the file LICENSE.md.
+ *  It is also available through the world-wide-web at this URL:
+ *  https://opensource.org/licenses/AFL-3.0
+ *  If you did not receive a copy of the license and are unable to
+ *  obtain it through the world-wide-web, please send an email
+ *  to tech@202-ecommerce.com so we can send you a copy immediately.
+ *
+ *  @author    202 ecommerce <tech@202-ecommerce.com>
+ *  @copyright Since 2019 Shopping Feed
+ *  @license   https://opensource.org/licenses/AFL-3.0  Academic Free License (AFL 3.0)
+ */
+
+namespace ShoppingfeedAddon\OrderImport\Rules;
+
+use ShoppingFeed\Sdk\Api\Order\OrderResource;
+use ShoppingfeedAddon\OrderImport\RuleAbstract;
+use ShoppingfeedClasslib\Extensions\ProcessLogger\ProcessLoggerHandler;
+use Tools;
+
+class ManomanoMissingCarrier extends RuleAbstract
```

Comment:
The goal is to manage a generic rules. All market place that not supply a carrier should have a default carrier named "standard". Please note the order of rules is important because there is an existing Zalendo rules that manage empty carrier with relais ID.

Reply:
The rule is renamed to MissingRule. It's applicable if the carrier is missing (restriction by the marketplace is removed). The rule performing is moved before ZolandoCarrier